os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -13,20 +13,25 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  ******************************************************************************/
+
 #include <osdep_service.h>
 #include <stdio.h>
 #include "rtl8721d_cache.h"
+#include <tinyara/sched.h>
+#include "../../../kernel/sched/sched.h"
```

Comment:
Could you let me know why you need this header?
We worry about using kernel resources from outside.

Reply:
because we need to get the info of a task, you can refer to check vTaskStackAddr, vTaskStackSize & vTaskName.